### DISCUSSION TITLE
Cache specific resolver results

### DISCUSSION THREAD

**Discussion by ziimakc** [mercurius-js/mercurius#839] [Q&A] [answered]
I would like to skip heavy count operations for pagination results, for example total count of users, as it's doesn't need to be accurate. Currently I do it manually, but is it possible to cache it with mercurius? `mercurius-js/cache` seems to work only for queries.

```
 UsersConnection: {
    total: async () => db.users.count()  // cache this for some time?
}
```

**Comment by mcollina** [comment, marked as answer]
it works only for queries. You'll need to use https://github.com/mcollina/async-cache-dedupe directly.
